src/win/util.c

```diff
@@ -484,7 +484,7 @@ uint64_t uv__hrtime(double scale) {
    * performance counter interval, integer math could cause this computation
    * to overflow. Therefore we resort to floating point math.
    */
-  return (uint64_t) ((double) counter.QuadPart * hrtime_interval_ * scale);
+  return (uint64_t) (counter.QuadPart * (uint64_t) scale / hrtime_interval_);
```

Comment:
> That's why I suggested to move the double cast to the end of the computation - `(counter.QuadPart / hrtime_interval_) * scale`

Oops, looks like I misunderstood that. Maybe this would be an acceptable fallback if integers absolutely do not work?